focus_converter_base/focus_converter/conversion_configs/aws/D013_S001.yaml

```diff
@@ -0,0 +1,4 @@
+plan_name: convert line_item_availability_zone to AvailibityZone
```

Comment:
It appears throughout this commit that Availability is spelled incorrectly, is that intentional for some field length constraint?